remotetracker/tracker.go

```diff
@@ -0,0 +1,15 @@
+package remotetracker
+
+// Tracker represents a remote issue tracker
+// for e.g github, trello, jira, bugzilla
+type Tracker interface {
+	Fetch(query string) error
+	Import() error
+}
+
+type Item struct {
```

Comment:
This should change to map[string]interface{}. We want the raw remote workitem data so we can map it directly.